contributors/guide/README.md

```diff
@@ -67,6 +68,10 @@ Have you ever wanted to contribute to the coolest cloud technology? We will help
 
 Please be aware that due to the large number of issues our triage team deals with, we cannot offer technical support in GitHub issues. If you have questions about the development process, feel free to jump into our [Slack Channel](http://slack.k8s.io/) or join our [mailing list](https://groups.google.com/forum/#!forum/kubernetes-dev). You can also ask questions on [ServerFault](https://serverfault.com/questions/tagged/kubernetes) or [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes). The Kubernetes team scans Stack Overflow on a regular basis, and will try to ensure your questions don't go unanswered.
 
+## Learning Golang Development
+
+Not all contributions to the project require knowledge of golang, but at the point you're delving into the language and development environment the [go code](go-code.md) document has a list of recommended reading, tools and tips and the [coding conventions](coding-conventions.md) document gives more kubernets specific guidance.
+
```

Comment:
typo: kubernetes

Reply:
will fix!

Reply:
I don't think it hurts to keep it for now, and on 1.0 we ask for feedback from the developer community on how relevant they feel this section is today.